src/main/java/com/github/sgov/server/service/WorkspaceService.java

```diff
@@ -218,21 +222,18 @@ public List<Workspace> findAllInferred() {
     /**
      * Removes vocabulary context from given workspace.
      *
-     * @param workspaceId         Uri of a workspace.
-     * @param vocabularyContextId Uri of a vocabulary context.
+     * @param workspaceId        Uri of a workspace.
+     * @param vocabularyFragment String of a vocabulary context UUID.
      */
-    public VocabularyContext removeVocabulary(URI workspaceId, URI vocabularyContextId) {
+    public VocabularyContext removeVocabulary(URI workspaceId, String vocabularyFragment) {
```

Comment:
And all method signatures within WorkspaceService uses URI only this method uses vocabularyFragment. Please, consider, not sure if it is possible to avoid ...

Reply:
The problem is that MC only sends UUID of workspace and vocabulary so the whole URI is not available. Before we could recreate the URI just from the UUID, but now we have `<vocabulary URI>/verze/<context UUID>` where `<vocabulary URI>` is not known.

Reply:
uff .. that is little wird, but ok